Sources/PointFree/Episode/Show.swift

```diff
@@ -896,25 +826,22 @@ let transcriptBlockView = View<Episode.TranscriptBlock> { block -> Node in
         `class`([outerVideoContainerClass, Class.margin([.mobile: [.topBottom: 2]])]),
         style(outerVideoContainerStyle)
       ],
-      [
-        video(
-          [
-            `class`([innerVideoContainerClass]),
-            controls(true),
-            playsinline(true),
-            autoplay(false),
-            Html.poster(poster),
-            style(objectFit(.cover))
-          ],
-
-          sources.map { source(src: $0) }
-        )
-      ]
+      video(
+        [
+          `class`([innerVideoContainerClass]),
+          controls(true),
+          playsinline(true),
+          autoplay(false),
+          Html.poster(poster),
+          style(objectFit(.cover))
+        ] as [Attribute<Tag.Video>],
+        ChildOf.fragment(sources.map { source(src: $0) })
```

Comment:
Also needed the additional `ChildOf` here to disambiguate for some reason. The prefix `...` didn't have that problem, strangely enough.